app/blog/[...puckPath]/page.tsx

```diff
@@ -0,0 +1,111 @@
+import { ComponentData, Data } from "@measured/puck";
+import { Render } from "@measured/puck/rsc";
+import { Metadata } from "next";
+import { notFound } from "next/navigation";
+
+import content from "../../../content.json";
+import resolvePuckPath from "../../../lib/resolve-puck-path";
+import { supabase } from "../../../lib/supabase";
+import config from "../../../puck.config";
+
+const { blogPostDescription, openGraphLocale, siteUrl, title } = content;
+
+const getPageRes = (path: string) =>
+  supabase.from("puck").select("*").eq("path", path).maybeSingle();
+
+export { viewport } from "../../page";
+
+export const dynamic = "force-static";
+
+export async function generateMetadata({
+  params,
+}: {
+  params: { puckPath: string[] };
+}): Promise<Metadata> {
+  const blogPath = "/blog";
+  const { path } = resolvePuckPath(params.puckPath);
```

Comment:
We might be able to `resolvePuckPath` soon because we have a distinct edit route